src/components/AgreementLinesFieldArray/AgreementLineField.js

```diff
@@ -38,6 +38,33 @@ export default class AgreementLineField extends React.Component {
     resource: {},
   }
 
+  validateDateOrder = (value, allValues, _props, name) => {
+    if (value) {
+      let activeFrom;
+      let activeTo;
+
+      if (name.indexOf('activeFrom') >= 0) {
+        activeFrom = new Date(value);
+        activeTo = new Date(get(allValues, name.replace('activeFrom', 'activeTo')));
+      } else if (name.indexOf('endDate') >= 0) {
+        activeFrom = new Date(get(allValues, name.replace('activeTo', 'activeFrom')));
+        activeTo = new Date(value);
+      } else {
+        return undefined;
+      }
+
+      if (activeFrom >= activeTo) {
+        return (
+          <div data-test-error-end-date-too-early>
```

Comment:
Maybe throw in a test case to validate that this is working?

Reply:
I think test for this would look out of place here, as we are depending on the PO lines tests here to test this functionality. A better idea would be to enforce this validation on the start and end dates while creating an agreement, and writing a test for it? That way we would have this validation logic covered in both agreements and licenses.

Reply:
I don't understand what PO lines have to do with this? I don't really care where the test is, but yeah I agree with you it makes sense to put it in the `agreement-crud` (or maybe actually the `basket`) test.

(In case it's confusing, I only put this comment down on this line just because I saw there were `data-test-` attributes that we could use to test if this validator is working correctly.)

Reply:
Yeah, I meant to say I didn't add the test in the PO lines tests, but that test functionality would run only when the PO lines tests run, as we are calling the _createAgreement_ function from there with a basket id passed to it, where then we populate the date fields.

Reply:
Add the validation test from within PO line as it was the easiest place to put in for this specific
functionality this PR covers.